### ISSUE TITLE
Call for maintainer of this package

### ISSUE THREAD

**Issue by tk0miya** [sphinx-contrib/domaintools#6]
Unfortunately, @klorenz ; the author of this package does not have enough time to maintain this package. I'd like to say thank you to him for his great product. And we should keep to maintain this. For that purpose, I'd like to call new maintainer of this package. Is there somebody would like to be maintainer?

refs:
- https://github.com/sphinx-contrib/domaintools/pull/4
- https://github.com/pypa/pypi-support/issues/22

**Comment by klorenz** [comment]
Hi @tk0miya, 

I just added a merge request myself (also updated my contact information).  Although I indeed am not the maintainer you are looking for, I could at least work through current merge-requests.

Regards, Kiwi

**Comment by klorenz** [comment]
Hi @tk0miya ,
I thought about it ... before you have none, who maintains this package, I can do it.

Regards, Kiwi

**Comment by n-peugnet** [comment]
@tk0miya: what would you think about maintaining this (or something similar) as a built-in sphinx API? I did not test it myself yet but it seems very useful.

From what I see in the comments of [the most important file](https://github.com/sphinx-contrib/domaintools/blob/master/sphinxcontrib/domaintools/__init__.py) it is mostly derived directly from Sphinx's Std domain's code, with additional parameters.

IMO allowing users to easily create custom domains would really highlight this feature of Sphinx.
I mean, I was interested by https://github.com/sphinx-doc/sphinx/pull/10517, especially with `pair` indices. But then I realised that a domain would be a perfect match for my use case. So a simple API for quickly creating a domain with a few directives would be really helpful.

**Comment by n-peugnet** [comment]
Never mind, I just found [Sphinx.add_object_type](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_object_type) which does exactly what I'm trying to achieve. It would be cool to highlight this function a little bit more in the "Domains" page of Sphinx's documentation, as I only found it mentioned in [the :any: role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-any).

**Comment by eric-wieser** [comment]
I think #15 answers this question? Namely, that @ktbarrett would be able to maintain this.

**Comment by ktbarrett** [comment]
Yeah I already have updated versions of `domaintools`, `makedomain`, and `prettyspecialmethods` I'm maintaining for a couple projects. It'd be no problem for me to take these over and spruce them up.

**Comment by eric-wieser** [comment]
I've granted you access to `prettyspecialmethods`
